recipes/batchspawner/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: batchspawner
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/batchspawner/batchspawner-{{ version }}.tar.gz
+  sha256: 9bae72f7c1bd9bb11aa58ecc3bc9fae5475a10fdd92dc0c0d67fa7eb95c9dd3a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+
+requirements:
+  host:
+    # only targets unix systems, imports `pwd`
+    - __unix
+    - pip
+    - python >=3.6
```

Comment:
Upstream has `>=3.3`. Up to you which one to use.

Ref: https://github.com/jupyterhub/batchspawner/blob/f4659678061dacf5837fb15ce340230dca18f3ca/setup.py#L43

Reply:
fair!